c/httpclient.c

```diff
@@ -1049,6 +1053,9 @@ int httpClientSessionReceiveNativeLoop(HttpClientContext *ctx, HttpClientSession
       bytesRead = socketRead(session->socket, buf, sizeof(buf), &bpxrc, &bpxrsn);
       if (bytesRead < 1) {
         HTTP_CLIENT_TRACE_VERBOSE("http client nativeLoop socket read error rc=%d, rsn=0x%x\n", bpxrc, bpxrsn);
+        if (bpxrc == 0x44E){
```

Comment:
I think we want to check for the time-out as well, otherwise it'll be unclear if this call has succeeded or timed out.

Something like this maybe:
```c
      if (rc == EWOULDBLOCK && rsn == JRTIMEOUT) {
        return RC_SEND_TIMEOUT;
      }
```